packages/json/src/json/convert-to-object.ts

```diff
@@ -1,6 +1,55 @@
+/**
+ * Converts a Map (back) to an object. Together with `convertToMap()`,
+ * you can update JSONs.
+ *
+ * @param object
+ *
+ * A Map.
+ *
+ * @return
+ *
+ * The Map as an object. (The object keys are sorted in alphabetical
+ * order.)
+ *
+ * @example
+ *
+ * Remove dependencies (if they exist) from `package.json`.
+ *
+ * ```ts
+ * const dependencies = convertToMap(packageJson['dependencies']);
+ *
+ * const packagesToDelete = [
+ *   '@embroider/macros',
+ *   'ember-auto-import',
+ *   'ember-cli-babel',
+ *   'ember-cli-htmlbars',
+ * ];
+ *
+ * packagesToDelete.forEach((packageName) => {
+ *   dependencies.delete(packageName);
+ * });
+ *
+ * packageJson['dependencies'] = convertToObject(dependencies);
+ * ```
+ *
+ * @example
+ *
+ * Configure `tsconfig.json` in an Ember app.
+ *
+ * ```ts
+ * const compilerOptions = convertToMap(tsConfigJson['compilerOptions']);
+ *
+ * compilerOptions.set('paths', {
+ *   [`${appName}/tests/*`]: ['tests/*'],
+ *   [`${appName}/*`]: ['app/*'],
+ *   '*': ['types/*'],
+ * });
+ *
+ * tsConfigJson['compilerOptions'] = convertToObject(compilerOptions);
+ * ```
+ */
 export function convertToObject(map = new Map()) {
   const sortedMap = new Map([...map.entries()].sort());
 
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
I had added this line in [#48](https://github.com/ijlee2/codemod-utils/pull/48/commits/15b8ab459649f296b46c5c24e7bc9f6ea4d0037b#diff-ee0fd92ee7d75b6662bdac034c80c4f64223e6380241b68245ecd7a65e3764baR4) because I tried to update the shared configuration to extend `@typescript-eslint/recommended-type-checked`.

Now that the shared configuration is back to extending `@typescript-eslint/recommended`, we can remove this line.